### PR TITLE
'delia-introduce'

### PR DESCRIPTION
**Pull request by delia-pixel**
### This PR add delia pixel to the Osscameroon community

and it is linked to #16

### CODE REVIEW

File: content/post/contributors/patricia_tchuegno.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+title: "Delia_pixel"
+date: 2022-01-04T13:46:09+01:00
+authors: ["Patricia Tchuegno"]
+---
+
+Hello, i'm Patricia Tchuegno, Sofware developper. I'm passionated of making beautiful websites. That's why i usually work on the front end side. But i also like writing algoritm and solving problems. So i can work on the backend side also. I have skills in digital marketing and want to learn more about Data Science. I used to code with AngularJs, ReactJs, Html, css, Javascript, Jquery, Bootstrap, Laravel, etc...
```
Comment (RMPR) [inline content/post/contributors/patricia_tchuegno.md:7]:
```suggestion
Hello, I'm Patricia Tchuegno, Sofware developper. I'm passionate about making beautiful websites. That's why I usually work on the front end side. But I also like writing algorithms and solving problems. So I can work on the back end side also. I have skills in digital marketing and want to learn more about Data Science. I used to code with AngularJs, ReactJs, HTML, CSS, Javascript, Jquery, Bootstrap, Laravel, etc.
```

File: content/post/contributors/patricia_tchuegno.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+By now i really want to perform my skills in programming to add my value and have more experience.
```
Comment (RMPR) [inline content/post/contributors/patricia_tchuegno.md:8]:
```suggestion
By now I really want to perform my skills in programming to add my value and have more experience.
```

> By now

You meant "But now"?

> perform my skills

You meant "perfect" maybe?

File: content/post/contributors/patricia_tchuegno.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+---
+title: "Delia_pixel"
+date: 2022-01-04T13:46:09+01:00
+authors: ["Patricia Tchuegno"]
+---
+
+Hello, I'm Patricia Tchuegno, Sofware developper. I'm passionate about making beautiful websites. That's why I usually work on the front end side. But I also like writing algorithms and solving problems. So I can work on the back end side also. I have skills in digital marketing and want to learn more about Data Science. I used to code with AngularJs, ReactJs, HTML, CSS, Javascript, Jquery, Bootstrap, Laravel, etc.
+By now I really want to perform my skills in programming to add my value and have more experience.
```
Comment (RMPR) [inline content/post/contributors/patricia_tchuegno.md:8]: > By now I really want to perform my skills in programming to add my value and have more experience.

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;^
This doesn't make much sense, you meant "perfect" maybe?

**Review by RMPR — COMMENTED** [review]
You probably also want to delete the `.hugo_build.lock` file so it doesn't appear as changed in the tree `git rm --cached .hugo_build.lock`

**Review by delia-pixel — COMMENTED** [review]
Here are the changes. @RMPR

**Review by elhmn — APPROVED** [review]

**Comment by elhmn** [comment]
@RMPR you can merge this when you are happy 🚀

**Pull request merged**